### ISSUE TITLE
[Suggestion] Shift Focus to Flight Controls/Feel

### ISSUE THREAD

**Issue by esauftgz** [tsunamayo/Starship-EVO#2545]
A few of us on the discord were discussing about how flight in its current state does not feel good. I believe that with improved flight controls, it should further encourage people to play the game, and even possibly encourage more community activities like fight clubs or races.

Here's the major issues:

1. Deadzone: Maneuvering, especially during a quick shift between opposite directions just feels too sluggish and imprecise, especially when you can go from not pitching much to pitching at full speed without much transition.

2. Cruise control: Having the ability to choose thruster output on a slider like in Elite Dangerous would help in control and maneuvers.

3. Precision mode: Something opposite of the current boost mode that allows for minimal speed would help a lot in docking and avoiding high speed collisions.

**Comment by SovietSponge** [comment]
One of the problems I have with the controls right now on top of what esau has already suggested is the fact that you just turn infinitely without getting your cursor back to the center of your screen instead of the ship pointing where the cursor points so it eventually centers it, sort of like in war thunder or elite dangerous. It might not be very easy to understand, I just don't know how else to describe it so I apologise for any potential difficulties.

**Comment by Tezlz** [comment]
Something like this has already been suggested a couple times before such as in #2024 and #2483 but considering how important the "flying-feel" is to the game it doesn't really hurt to have more posts about it to clarify how much it matters.

**Comment by tsunamayo** [comment]
Yes,  it is on my to-do list for today actually. Thanks

**Comment by tsunamayo** [comment]
@esauftgz @Tezlz @SovietSponge Does it feels better in latest update? I updated the mouse control, it should be much tighter.

**Comment by Tezlz** [comment]
Yes it feels a lot more tight and precise now, especially with a more visible deadzone so you know where to center to stop spinning. I can't think of much that could be done to improve it as it's pretty good as is.
